aad/spring-security/reactive/webflux/oauth2/spring-cloud-gateway/client/src/main/java/com/azure/spring/sample/activedirectory/reactive/webflux/oauth2/gateway/client/controller/ResourceServer1Controller.java

```diff
@@ -0,0 +1,30 @@
+package com.azure.spring.sample.activedirectory.reactive.webflux.oauth2.gateway.client.controller;
+
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.annotation.RegisteredOAuth2AuthorizedClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import static org.springframework.security.oauth2.client.web.reactive.function.client.ServerOAuth2AuthorizedClientExchangeFilterFunction.oauth2AuthorizedClient;
+
+@RestController
+public class ResourceServer1Controller {
+
+    private final WebClient webClient;
+
+    public ResourceServer1Controller(WebClient webClient) {
+        this.webClient = webClient;
+    }
+
+    @GetMapping("/resource-server-1")
+    public Mono<String> hello(@RegisteredOAuth2AuthorizedClient("client-1-resource-server-1") OAuth2AuthorizedClient client1ResourceServer1) {
```

Comment:
I think it's too long here for code, we don't have checkstyle now, but I think we will.
The code line better not longer than 100.

Reply:
Updated.